index.html

```diff
@@ -1053,9 +1059,6 @@ <h3>Data delivery</h3>
             <li>
               If |observer|.{{PressureObserver/[[AfterPenaltyRecordMap]]}}[source] [=map/exists]:
```

Comment:
I would say that when we are in penalty we ignore all data (basically like we unobserved), ie. don't send events and also don't need to count changes. 

When we are out of penalty, we will start receiving events again like was it a new call to observe. If the system is still being abused, we will after a short while go into penalty mode again.

Maybe it is easier to spec/implement this as actually unobserving and scheduling a new internal observe call as soon as the penalty time is over?

Reply:
It s working fine, as you describe it, I wouldnt change the specs anymore.
One thing still when you say.
>> would say that when we are in penalty we ignore all data (basically like we unobserved), ie. don't send events and also don't need to count changes.

When we are in Penalty, we are not fully ignoring the data. We still need to save the last received state during the penalty, so that user can fall back on its feet. This is the reason for AfterPenaltyRecordMap

agree?

Reply:
Agree, that is fine. I thought about it just after I sent the reply.

Will you fix the remaining comments?

Reply:
Unless mistaken I see only 1 left.

about the Continue as a separate step. Doing it right now